data-service/src/main/java/com/amazonaws/blox/dataservice/repository/EnvironmentRepositoryDDB.java

```diff
@@ -366,4 +366,21 @@ public void deleteEnvironmentRevision(@NonNull final EnvironmentRevision environ
           e);
     }
   }
+
+  @Override
+  public void deleteEnvironment(@NonNull final EnvironmentId environmentId)
+      throws InternalServiceException {
+    try {
+      dynamoDBMapper.delete(
+          EnvironmentDDBRecord.withKeys(
+              environmentId.generateAccountIdCluster(), environmentId.getEnvironmentName()),
+          SaveBehavior.CLOBBER.config());
```

Comment:
Why need a `SaveBehavior.CLOBBER.config()` instead of the default config?  The SaveBehavior probably be only useful for save and update?

Reply:
I was thinking about using the default one, but I see that the "deleteEnvironmentRevision" also uses this SaveBahavior. I can just change both back to the default

Reply:
After reading about the SaveBehavior I think it should be used for deleteEnvironmentRevision but not deleteEnvironment. The SaveBehavior.CLOBBER.config() is to delete when the version is out of date. We don't need this feature for deleteEnvironment I think.